### DISCUSSION TITLE
Authorization issue

### DISCUSSION THREAD

**Discussion by msgnoki** [huggingface/autotrain-advanced#767] [General]
Hi,

I am experiencing an issue when launching AutoTrain locally. The logs indicate the following:

vbnet
Copier le code
INFO     | 2024-09-18 16:15:46 | autotrain.app.ui_routes:<module>:32 - Starting AutoTrain...
INFO     | 2024-09-18 16:15:53 | autotrain.app.ui_routes:<module>:298 - AutoTrain started successfully
INFO     | 2024-09-18 16:15:53 | autotrain.app.app:<module>:13 - Starting AutoTrain...
INFO     | 2024-09-18 16:15:53 | autotrain.app.app:<module>:23 - AutoTrain version: 0.8.19
INFO     | 2024-09-18 16:15:53 | autotrain.app.app:<module>:24 - AutoTrain started successfully
INFO:     Started server process [12774]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:8080 (Press CTRL+C to quit)
INFO:     127.0.0.1:48432 - "GET / HTTP/1.1" 307 Temporary Redirect
INFO:     127.0.0.1:48432 - "GET /ui/ HTTP/1.1" 401 Unauthorized
INFO:     127.0.0.1:48432 - "GET /favicon.ico HTTP/1.1" 404 Not Found

I don’t understand why I’m getting a 401 Unauthorized error and a 404 Not Found error. I would appreciate any assistance in troubleshooting this issue.

Thank you for your help!

**Comment by msgnoki** [comment]
{"detail":"Invalid or expired token"} on browser

**Comment by msgnoki** [comment]
RTFM, forgot to export my HF write token

**Comment by menacing-american-patriot** [comment]
'export' is not recognized as an internal or external command,
operable program or batch file.
